library_panel.go

```diff
@@ -101,22 +101,34 @@ func (c *Client) LibraryPanels() ([]LibraryPanel, error) {
 
 // LibraryPanelByUID gets a library panel by UID.
 func (c *Client) LibraryPanelByUID(uid string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/%s", uid))
+	resp := &LibraryPanelCreateResponse{}
+	path := fmt.Sprintf("/api/library-elements/%s", uid)
+
+	err := c.request("GET", path, nil, nil, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.Result, err
 }
 
 // LibraryPanelByName gets a library panel by name.
 func (c *Client) LibraryPanelByName(name string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/name/%s", name))
-}
+	var resp struct {
+		Result []LibraryPanel `json:"result"`
+	}
+	path := fmt.Sprintf("/api/library-elements/name/%s", name)
 
-func (c *Client) panel(path string) (*LibraryPanel, error) {
-	resp := &LibraryPanelCreateResponse{}
 	err := c.request("GET", path, nil, nil, &resp)
 	if err != nil {
 		return nil, err
 	}
 
-	return &resp.Result, err
+	if len(resp.Result) != 1 {
+		return nil, fmt.Errorf("error: expected 1 panel from GET library panel by name, got: %v", resp.Result)
```

Comment:
Nit: no need to add the error prefix:

```suggestion
		return nil, fmt.Errorf("expected 1 panel from GET library panel by name, got: %v", resp.Result)
```